text/src/autogluon/text/automm/utils.py

```diff
@@ -1221,3 +1225,38 @@ def turn_on_off_feature_column_info(
                 per_model_processor.requires_column_info = flag
 
     return data_processors
+
+
+def try_to_infer_pos_label(
+        data_config: DictConfig,
+        label_encoder: LabelEncoder,
+        problem_type: str,
+):
+    """
+    Try to infer positive label for binary classification, which is used in computing some metrics, e.g., roc_auc.
+    If positive class is not provided, then use pos_label=1 by default.
+    If the problem type is not binary classification, then return None.
+
+    Parameters
+    ----------
+    data_config
+        A DictConfig object containing only the data configurations.
+    label_encoder
+        The label encoder of classification tasks.
+    problem_type
+        Type of problem.
+
+    Returns
+    -------
+
+    """
+    if problem_type != BINARY:
+        return None
+
+    pos_label = OmegaConf.select(data_config, "pos_label", default=None)
+    if pos_label:
```

Comment:
Will it be problematic if pos_label is 0!

Reply:
Thanks. Fixed.